app/controllers/gift_requests_controller.rb

```diff
@@ -7,20 +7,23 @@ class GiftRequestsController < ApplicationController
 
   def new
     @back_route = wishlist_path(params[:id])
-
     @gift_request = GiftRequest.new
-    render "gift_request", locals: {list_id: params[:id]}
+    @list_name = List.find(params[:id]).name
+
+    render "gift_request"
   end
 
   def edit
     @back_route = wishlist_path(params[:id])
-
-    render "gift_request", locals: {list_id: params[:id]}
+    
+    render "gift_request"
```

Comment:
I don't like it - I prefer views to stay as dumb as possible for consistency's sake - but it's an argument of preference at this point, so go ahead and :ship: